fury/utils.py

```diff
@@ -753,6 +753,12 @@ def numpy_to_vtk_matrix(array):
 
 def get_bounding_box_sizes(actor):
     """Get the bounding box sizes of an actor."""
+    from fury.ui import UI  # to avoid circular import
+
+    if isinstance(actor, UI):
+        width, height = actor.size
+        return (width, height, 0)
```

Comment:
I recommend avoiding importing `UI` in `utils`. `Utils` is dedicated to `actor`. 

I understand that you want only one function to get the size so you should define this new function in the layout class

Reply:
Hey @skoudoro, thanks for the suggestion/reminder. I have removed the size calculation from `utils` and added it to `GridLayout` class. Unfortunately, importing `UI` in `layout` module by `from fury.ui.core import UI` doesn't work either for me. I get the following error:-
```python
File "D:\pythonProjects\fury-event\fury\fury\layout.py", line 5, in <module>        
    from fury.ui.core import UI
  File "D:\pythonProjects\fury-event\fury\fury\ui\__init__.py", line 3, in <module>   
    from fury.ui.containers import *
  File "D:\pythonProjects\fury-event\fury\fury\ui\containers.py", line 12, in <module>
    from fury.actor import grid
  File "D:\pythonProjects\fury-event\fury\fury\actor.py", line 2135, in <module>
    class Container(object):
  File "D:\pythonProjects\fury-event\fury\fury\actor.py", line 2152, in Container
    def __init__(self, layout=layout.Layout()):
AttributeError: module 'fury.layout' has no attribute 'Layout'
```